packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/delete-dialog.tsx

```diff
@@ -2,16 +2,48 @@ import React from 'react';
 import { Dialog, Text } from '@deriv/components';
 import { localize } from '@deriv/translations';
 import { connect } from 'Stores/connect';
+import { getSavedWorkspaces } from '@deriv/bot-skeleton';
+import LZString from 'lz-string';
+import localForage from 'localforage';
 
 type TDeleteDialog = {
     is_running: boolean;
     is_delete_modal_open: boolean;
-    onToggleDeleteDialog: (type: string, param: boolean) => void;
+    onToggleDeleteDialog: (param: boolean) => void;
+    removeBotStratagy: (param: string) => void;
+    selected_strategy_id: string;
+    setStrategies: (param: string[]) => void;
+    setDashboardStrategies: (param: string[]) => void;
+    dashboard_strategies: [];
 };
 
-const DeleteDialog = ({ is_delete_modal_open, is_running, onToggleDeleteDialog }: TDeleteDialog) => {
+const DeleteDialog = ({
+    is_delete_modal_open,
+    is_running,
+    selected_strategy_id,
+    onToggleDeleteDialog,
+    setDashboardStrategies,
+    dashboard_strategies,
+}: TDeleteDialog) => {
+    const removeBotStratagy = async (strategy_id: string) => {
```

Comment:
typo in strategy 
also please check for other typo

Reply:
done ✅